examples/hellohttp/e2e-test.sh

```diff
@@ -42,20 +42,43 @@ function EXPECT_CONTAINS() {
 }
 
 if [[ -z "${1:-}" ]]; then
-  echo "Usage: $(basename $0) <kubernetes namespace> <language ...>"
+  echo "ERROR: None of execution type, kubernetes namspace and languages is provided!"
```

Comment:
are these validations identical to the ones in examples/hellogrpc/e2e-test.sh?
If so, lets create a util.sh somewhere that exports a validation function

Reply:
Done